packages/wallet/api/package.json

```diff
@@ -11,7 +11,7 @@
     "lint": "run-s --continue-on-error lint:*",
     "lint-fix": "yarn lint:eslint --fix",
     "lint:types": "tsc",
-    "lint:eslint": "eslint ."
+    "lint:eslint": "eslint --report-unused-disable-directives --fix ."
```

Comment:
oops I missed this one

Reply:
Pretty sure that `--fix` doesn't belong there.

Reply:
Right if tests modify source, you will get CI to complain (once I get around to it), which would cause people to complain that they don't know why CI is complaining.

Reply:
No changes to `package.json` were intended. What I meant is I missed removing the change.